docs/tables.rst

```diff
@@ -137,12 +137,20 @@ Visualizations
     :toctree: _autosummary
 
     Table.plot
+    Table.iplot
```

Comment:
If our preferred way of users using the interactive plots is by using the interactive method, then I'm not sure that we should be documenting the methods straight. Rather, merge this documentation into the non-interactive method one's - and that shouldn't add much extra stuff, given that the method signatures should be identical.

Reply:
changed.